<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuCaWidget</class>
 <widget class="QWidget" name="MenuCaWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>540</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>系统设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="bottomLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>470</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>【菜单】退出当前页面</string>
   </property>
  </widget>
  <widget class="QLabel" name="usrLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>用户属性：</string>
   </property>
  </widget>
  <widget class="QLabel" name="cardLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>智能卡属性：</string>
   </property>
  </widget>
  <widget class="QLabel" name="pairLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>机卡配对：</string>
   </property>
  </widget>
  <widget class="QLabel" name="usrDataLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>普通用户</string>
   </property>
  </widget>
  <widget class="QLabel" name="cardDataLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>71180200714C6C36</string>
   </property>
  </widget>
  <widget class="QLabel" name="pairDataLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>未配对</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
